### PR TITLE
Schools Edit Page and tests, + Minor CreatePageTest error fixed

### LINKED ISSUE #32: School Edit Page
part of Epic #4

### PR DESCRIPTION
**Pull request by SR33G33**
Closes #32 
Edit Page for Admins is visible in the Schools tab when edit button is pressed

image

allows user to edit successfully
TODO: Still allows users to edit abbrev which cause some funky stuff to happen where it overwrites another table entry if the new abbrev matches an old one

image

image

This branch is currently deployed to: https://proj-organic-sr33g33-dev.dokku-12.cs.ucsb.edu/

*Note if the deployed app looks like an old one I might just have pushed a different branch to debug, shoot me a message on slack and I can deploy the correct branch

### CODE REVIEW

**Review by jason-rodrigues — COMMENTED** [review]
image
image
Term RegEx doesn't seem to be enforced on update

**Comment by SR33G33** [comment]
These are kind of backend issues, Ill make a new PR to fix these after this one is merged

TODO: Make the update check that the termDescription matches new TermRegex
TODO: Make sure abbrev cant be edited in update

**Review by ElijahCanderson — DISMISSED** [review]
LGTM

**Comment by pconrad** [comment]
The create school button leads to an error message for me on the dokku deployment linked in the PR.

I can't merge this unless/until I can test it.

**Comment by SR33G33** [comment]
redeployed app and seems to be working on my end, I pushed the most updated working version that has edit and delete and create functionality

note: still has the edit page bugs that I pointed out in the edit page PR which will be fixed in a different PR

File: frontend/src/tests/pages/SchoolCreatePage.test.js
Comment (pconrad) [inline frontend/src/tests/pages/SchoolCreatePage.test.js]:
```suggestion
                "termDescription": "quarter",
```

**Review by pconrad — CHANGES_REQUESTED** [review]
Let's clear up the confusion about the use case for these fields.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**
